crates/lightbeam/src/module.rs

```diff
@@ -563,15 +563,17 @@ pub fn translate_only(data: &[u8]) -> Result<TranslatedModule, Error> {
         let memories = section.get_memory_section_reader()?;
         let mem = translate_sections::memory(memories)?;
 
-        assert_le!(
-            mem.len(),
-            1,
-            "Multiple memory sections not yet unimplemented"
-        );
+        if mem.len() > 1 {
+            return Err(Error::Input(
+                "Multiple memory sections not yet unimplemented".to_string(),
```

Comment:
This was pre-existing, but we should fix it while here: "not yet unimplemented" is a double negative; it should say "not yet implemented".

Reply:
replaced ;)